.gitignore

```diff
@@ -1,51 +1,15 @@
-# Object files
-*.o
-*.ko
-*.obj
-*.elf
-
-# Precompiled Headers
-*.gch
-*.pch
-
-# Libraries
-*.lib
-*.a
-*.la
-*.lo
-
-# Shared objects (inc. Windows DLLs)
-*.dll
-*.so
-*.so.*
-*.dylib
-
-# Executables
-*.exe
-*.out
-*.app
-*.i*86
-*.x86_64
-*.hex
-*.initrd
-
-# Debug files
-*.dSYM/
-
-# Images
-*.iso
-*.bin
-
-# Intellij
-.idea/
-
-# VSCode
-.vscode/
-
-# Zig
-zig-cache
-
-# Build dir
-bin/*
-*.pyc
-__pycache__
+# Binaries
+**/*.elf
+**/*.bin
+**/*.o
+**/*.iso
```

Comment:
Are the "**" necessary?

Reply:
It so it can find these files in side other directories. So will cover all files with these extentions

Reply:
Alright, as long as this also ignores files in the project root, this is ok.